PHP/PHP Source.sublime-syntax

```diff
@@ -106,7 +106,7 @@ contexts:
     - include: comments
     - match: |-
         (?x)
-        \s*(array) # Typehint
+        \s*(array|callable|int|string) # Typehint
```

Comment:
What about `bool` and `float`? (http://php.net/manual/en/functions.arguments.php#functions.arguments.type-declaration)